nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw fix modal width on mobile viewport

# Description
No issue was created. Fixed a padding-left problem where the style was being written inline by bootstrap and causing the modal to be off-centered on mobile phone view.
**NOTE:** The branch for this Pull Request was built off the branch on Pull Request #140 . Checking this Pull Request and the items in the PR #140 will cause only this branch to need to be merged with Main. The other still needs to be reviewed, however.

## Type of change


# Testing Instructions
- Pull down branch and serve website in a browser
- Shrink browser to mobile viewport width then open developer modal by clicking a **Learn More** button. The modal should be centered on the screen.  


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
